### PR TITLE
Bug fix in sandbix start 

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
- Updated boilerplate 
-  Small bug fix in version compare until (Currently we have a condition that checks greater then equal but now we only check GreateThen.....This bug exist in version compare)
- Because we are mounting container file in localhost system that's why we are not able to change the permission of the file, But now we created a empty file and we only overwrite while mounting docker volume

## Type
- [x] Bug Fix
- [ ] Feature
- [ ] Plugin

## Are all requirements met?

- [ ] Code completed
- [ ] Smoke tested
- [ ] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
_How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -48,7 +52,7 @@ func IsVersionGreaterThan(version1, version2 string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return semanticVersion2.LessThanOrEqual(semanticVersion1), nil
+	return semanticVersion1.GreaterThan(semanticVersion2), nil
```
Comment (pmahindrakar-oss) [inline pkg/util/util.go:55]: Lets  add unit test with the bug symptoms  and before the change that test should fail and after change it should pass.

File: cmd/sandbox/start.go [hunk 2]
```diff
@@ -55,7 +55,7 @@ Run specific version of flyte, Only available after v0.14.0+
 Usage
 	`
 	k8sEndpoint                  = "https://127.0.0.1:30086"
-	flyteMinimumVersionSupported = "v0.14.0"
+	flyteMinimumVersionSupported = "v0.13.0"
```
Comment (pmahindrakar-oss) [inline cmd/sandbox/start.go:58]: Lets update the docs also to mention this if not already.

File: pkg/util/util.go (hunk 1 above)
Comment (evalsocket) [reply pkg/util/util.go:55]: ok

**Review by pmahindrakar-oss — APPROVED** [review]
